cli/config/configuration-qemu-virtiofs.toml.in

```diff
@@ -466,3 +466,17 @@ sandbox_cgroup_only=@DEFSANDBOXCGROUPONLY@
 # Supported experimental features:
 # (default: [])
 experimental=@DEFAULTEXPFEATURES@
+
+# Use VFIO driver for VFIO devices in the VM, as well as the host.
```

Comment:
Based on the description, I would call the variable `use_guest_driver_for_vfio_devices` or something like that. You are not enabling VFIO in the guest, you are changing how the driver is bound.

Reply:
`use_guest_driver_for_vfio_devices` is ambiguous, because it's not clear if you're using the guest's VFIO driver or the guest's "normal" driver for the device, which is precisely the issue.  Originally I wanted to call this `use_guest_native_driver_for_vfio` with the sense reversed (so true for the old behaviour, false for the new).  But for irritating reasons I couldn't figure out a sane way to add a boolean config option that defaulted to true.

Reply:
What about `unbind_guest_vfio_drivers`?

Reply:
That's even worse: we don't want to unbind the vfio drivers, we want to unbind the not-vfio drivers for the vfio devices.